docs/pages/guides/meshery-metrics.md

```diff
@@ -1,99 +1,195 @@
 
+In order to pull in these environment metrics, you can also manually configure Meshery to connect with your existing Grafana and/or Prometheus instances through the Meshery dashboard. Once they have been loaded and are displayed on the screen, you may also perform an *ad-hoc* test to check Meshery's connection status.
 
-## Istio + Minikube
+## Tutorial Guide
 
-In this scenario, Meshery has deployed Istio to a Kubernetes cluster running in Minikube. The service ports of*Grafana*and Prometheus need to be exposed in order for Meshery to connect to and interact with these visualization and metrics systems.
+Follow along with this tutorial to set up and integrate Grafana and Prometheus instances on top of your service mesh and run performance tests:
 
-### Get minikube ip
+* Deploy a service mesh and any available sample application
 
-Retreive the IP address of your Minikube cluster by executing:
-    
-```        
-    $ minikube ip
-    172.17.0.2
-```
+  For the purpose of this tutorial guide, we will be installing [Istio](/docs/service-meshesadapters/istio) and deploying the [BookInfo sample application](/docs/guides/sample-apps#bookinfo):
+
+    - [Install Istio](/docs/service-meshesadapters/istio) on Meshery in the `istio-system` namespace. 
+      <a href="/docs/assets/img/adapters/istio/istio-install.png">
+      <img style="width:300px;padding-top:5px;" src="/docs/assets/img/adapters/istio/istio-install.png" />
+      </a>
+    - [Deploy the BookInfo sample application](/docs/guides/sample-apps#to-deploy-a-sample-app-on-meshery) 
+
+      The [BookInfo](/docs/guides/sample-apps#bookinfo) sample app should be installed in the `default` namespace unless otherwise specified upon deployment of BookInfo.
 
-<i>Note: Istio is installed in `istio-system` namespace and the "BookInfo" sample app is installed in `default` namespace unless otherwise specified upon deployment of "BookInfo".
-</i>
+* Set up Grafana and/or Prometheus:
+1. [Connect Meshery to metric systems](#connect-meshery-to-metric-systems)
+2. [Connect Meshery to metric systems using Minikube](#Connect-Meshery-to-metric-systems-using-Minikube)
 
-### Expose Prometheus service
+* Expose the service metric ports - The service ports of Grafana and Prometheus need to be exposed in order for Meshery to connect to and interact with these visualizations and metrics systems.
+* Access the port assigned to the metric service
+* [Run Performance tests](#run-performance-tests)
 
-a) By default  *prometheus* service spec type is configured to *ClusterIP*. You can change it to **NodePort** by executing:
+
+### **Connect Meshery to metric systems**
```

Comment:
It's ideal for us not to include style here. If the <h3> tag isn't working across the board, let's consider a stylesheet update. Otherwise, can we remove the bold font weight?